composer.json

```diff
@@ -38,6 +38,7 @@
         "bacon/bacon-qr-code": "^1.0.3 || ^2.0.0",
         "christian-riesen/base32": "^1.3.2",
         "composer": "^2.0.0",
+        "cypht/cli-service": "*",
```

Comment:
Why do we define another package? It doesn't seem to be runnable without main cypht repo, so better stay as just a subdirecrory in this repo. No need for composer dependencies here.